src/Files.App/Files.App.csproj

```diff
@@ -39,6 +39,9 @@
         <DefineConstants>TRACE;RELEASE;NETFX_CORE;DISABLE_XAML_GENERATED_MAIN</DefineConstants>
         <Optimize>true</Optimize>
     </PropertyGroup>
+    <ItemGroup>
+      <None Remove="UserControls\Widgets\NetworkLocationsWidget.xaml" />
```

Comment:
The xaml files are included by default so we can remove this group

Reply:
Fixed.